.github/workflows/static_checks_etc.yml

```diff
@@ -34,15 +34,11 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run FOSSA scan and upload build data
-        # Fails on pull requests when using the API key secret.
-        # In order to run it on pull requests we would need to
-        # generate a push only token and specify that as plain
-        # text here:
-        #   https://github.com/fossa-contrib/fossa-action#push-only-api-token
-        if: github.ref == 'refs/heads/main'
-        uses: fossa-contrib/fossa-action@v2
+        if: steps.skip-workflow.outputs.skip-workflow == 'false'
+        uses: fossa-contrib/fossa-action@master
```

Comment:
Should we be explicit with `@v3`?

Reply:
We were. I was trying HEAD as a test to see if it was any better/different. It's not.